tests/unit/copydb.sh

```diff
@@ -40,12 +38,21 @@ find .
 
 pgopts="--single-transaction --no-psqlrc --expanded"
 
-for f in ./sql/*.sql
+for f in ./tests/*.{sql,sh}
```

Comment:
Could we keep the `sql` directory, like they do in Postgres, and add a new directory for the bash scripts, maybe `script` or something? Then instead of dealing with the `if [[ ... ]]` in the loop body, we would have two loops, first the SQL one I suppose, and then the bash/script specialized one. Also would make it easier to deal with DIFFOPTS which at the moment is generalized in the call to `diff` but initialized only in one case...